frontend/tailwind.config.js

```diff
@@ -0,0 +1,16 @@
+/** @type {import('tailwindcss').Config} */
+// module.exports = {
+//   content: [],
+//   theme: {
+//     extend: {},
+//   },
+//   plugins: [],
+// }
+
+module.exports = {
+  content: ["./src/pages/Demo/DemoTailwind.tsx"], // "./src/**/*.{js,jsx,tsx}", "./templates/index.html"
+  theme: {
+    extend: {},
+  },
+  plugins: [],
+};
```

Comment:
my suggestion is to disable the tailwind normalization's completely like this

```javascript
module.exports = {
  corePlugins: {
    preflight: false,
  },
  // ...
}
```

and then we hope that the Sass normalizations cover bases as we build new pages with tailwind :crossed_fingers:

Reply:
Hey @MattPereira thanks for the suggestion✨! I tried disabling the preflight but it will affect components that rely only on tailwind for styling. However, it is a quick way for devs to check whether certain style changes are caused by tailwind preflight. I added these lines to tailwind.config.js as comments for future reference.

Reply:
lol turned out to be a not good suggestion but I do appreciate the sentiment :sweat_smile: